src/e2e-test/java/io/cdap/plugin/utils/PluginUtils.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+/**
+ * Plugin - Utils.
+ */
+public class PluginUtils {
+  private static final Logger logger = LoggerFactory.getLogger(PluginUtils.class);
+  public static Properties pluginProperties = new Properties();
+  public static Properties errorMessageProperties = new Properties();
+
+  static {
+    try {
+      pluginProperties.load(new FileInputStream("src/e2e-test/resources/plugin_properties.properties"));
+      errorMessageProperties.load(new FileInputStream("src/e2e-test/resources/error_messages.properties"));
+    } catch (IOException e) {
+      logger.error("Error while reading file: " + e);
+    }
+  }
```

Comment:
This looks like common code that belongs in the framework.  Consider adding plugin util methods in https://github.com/cdapio/cdap-e2e-tests/blob/develop/src/main/java/io/cdap/e2e/utils/PluginPropertyUtils.java